utils/log.go

```diff
@@ -4,15 +4,16 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strconv"
 	"sync"
 )
 
-var out io.Writer = os.Stdout
-
 // LogLevel of quic-go
 type LogLevel uint8
 
 const (
+	logEnv = "QUIC_GO_LOG_LEVEL"
```

Comment:
I think we should make colissions as unlikely as possible, so I'd prefer our full name.